package.json

```diff
@@ -20,17 +22,22 @@
     "lodash": "4.17.15",
     "multilines": "1.0.2",
     "node-fetch": "2.6.0",
-    "probot": "9.2.20"
+    "probot": "9.6.4",
+    "probot-serverless-now": "^2.1.1"
```

Comment:
In this project, we should use fixed dependencies everywhere.

Reply:
I should have guessed when I saw the rest of the dependencies were fixed. Fixed all in https://github.com/opencollective/opencollective-bot/pull/237/commits/33b595fab1b303984b825667833a6a42041fdf70